atom/browser/atom_browser_main_parts.cc

```diff
@@ -223,8 +223,7 @@ void AtomBrowserMainParts::PreMainMessageLoopRun() {
 #if !defined(OS_MACOSX)
   // The corresponding call in macOS is in AtomApplicationDelegate.
   Browser::Get()->WillFinishLaunching();
-  std::unique_ptr<base::DictionaryValue> empty_info(new base::DictionaryValue);
```

Comment:
@ckerr this one was also funny

Reply:
there is also another instance at https://github.com/electron/electron/blob/master/atom/browser/mac/atom_application_delegate.mm#L65

Reply:
done